src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt

```diff
@@ -104,8 +108,20 @@ class ProtectLevelCalculator(
   ): Int =
     when {
       isFemale(crn) -> {
-        (getAdditionalFactorsAssessmentComplexityPoints(offenderAssessment) + getBreachRecallComplexityPoints(crn, convictions))
-          .times(2)
+        val additionalFactorsPoints = getAdditionalFactorsAssessmentComplexityPoints(offenderAssessment)
+        val breachRecallPoints = getBreachRecallComplexityPoints(crn, convictions)
+
+        val violenceArson = when {
```

Comment:
Small matters now but half of these values are suffixes with 'Points' and half are not. We should choose one, don't mind which.

Also half of the calculations are abstracted away in functions and half are not. Again we should choose one, and here I have a preference for abstraction on the grounds of readability.

An alternative might even be to wrap the three calculations based on the convictions into a single method? It might be difficult to name though

The root cause is that there is a great deal of conplexity in this class